lib/resources/checks.js

```diff
@@ -21,6 +21,10 @@ util.inherits(Checks, ResourceBase);
     return this._transmit('GET', null, options, null, callback);
   };
 
+  this.delete = function (id, callback) {
+    return this._transmit('DELETE', id, null, null, callback);
+  }
```

Comment:
Nit: there should be a semi colon at the end of this function expression (along with the `this.delete`s in the other files). The linter should've caught this... That makes me think that there's something amiss with how our linter is set up. I'll take a look to see what's up.